### DISCUSSION TITLE
Watching live on zdf.de will sooner or later freeze Firefox

### DISCUSSION THREAD

**Discussion by SvenMichaelKlose** [aclap-dev/video-downloadhelper#1331] [Help and Q&A]
### Your question

Where is the bug report section here, I'd ask. 8) However: ii's been like this for a couple of weeks.

As long as a channel is playing everything is fine. Freezes occur during breaks.

Best regards,
pixel

### Addon & CoApp version and details

Version 9.1.0.44 (Firefox 130.0.1  on Fedora 40)

**Comment by Mattmjs** [comment]
What does this have to do with the extension ?, if you're not planning to download something then disable it.

**Reply by SvenMichaelKlose** [reply]
Charming. Guess what I am doing already. Never mind. There'll be an alternative.
